prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -641,30 +641,48 @@ def prompt_processor() -> Any:
     return structured_output
 
 
-def simple_retriver(  # type:ignore
+def simple_or_subq_retriever(  # type:ignore
     output: dict[str, Any],
     doc_id: str,
     llm_helper: ToolLLM,
     llm_li: Optional[LLM],
     vector_index,
+    retrival_type: str,
```

Comment:
@harini-venkataraman Let's fix typo and rename all variables to `retrieval_*`, etc.